project_purchase_link/models/project_project.py

```diff
@@ -40,13 +40,10 @@ def _compute_purchase_info(self):
 
     def _compute_purchase_invoice_info(self):
         for project in self:
+            domain = project._get_purchase_invoice_domain()
```

Comment:
@acsonefho Why not doing the read group outside the loop and adding analytic_account_id in grouping ?